README.md

```diff
@@ -0,0 +1,795 @@
+            </tr>
+          </tbody>
+        </table>
+        Optional: Microsoft Entra ID (AAD) Role 'Directory readers' could be used instead of API permissions (more 'read' than required)
+      </td>
+    </tr>
+    <tr>
+      <td><b>D</b><br>Azure DevOps / Github Actions | ServicePrincipal</td>
+      <td>
+        <table>
+          <tbody>
+            <tr>
+              <th>Capability</th>
+              <th>API Permissions</th>
+            </tr>
+            <tr>
+              <td>Get Microsoft Entra ID (AAD)<br>Users</td>
+              <td>Azure DevOps Service Connection's <b>App registration</b><br>grant with <b>Microsoft Graph</b> permissions:<br>Application permissions / User / User.Read.All<br>&#x1F4A1; <a href="https://docs.microsoft.com/en-us/graph/api/user-get#permissions" target="_blank">Get user</a></td>
+            </tr>
+            <tr>
+              <td>Get Microsoft Entra ID (AAD)<br>Groups</td>
+              <td>Azure DevOps Service Connection's <b>App registration</b><br>grant with <b>Microsoft Graph</b> permissions:<br>Application permissions / Group / Group.Read.All<br>&#x1F4A1; <a href="https://docs.microsoft.com/en-us/graph/api/group-get#permissions" target="_blank">Get group</a></td>
+            </tr>
+            <tr>
+              <td>Get Microsoft Entra ID (AAD)<br>SP/App</td>
+              <td>Azure DevOps Service Connection's <b>App registration</b><br>grant with <b>Microsoft Graph</b> permissions:<br>Application permissions / Application / Application.Read.All<br>&#x1F4A1; <a href="https://docs.microsoft.com/en-us/graph/api/serviceprincipal-get#permissions" target="_blank">Get servicePrincipal</a>, <a href="https://docs.microsoft.com/en-us/graph/api/application-get#permissions" target="_blank">Get application</a></td>
+            </tr>
+            <tr>
+              <td>Get PIM Eligibility<br>SP/App</td>
+              <td>Service Principal's <b>App registration</b><br>grant with <b>Microsoft Graph</b> permissions:<br>Application permissions / PrivilegedAccess / PrivilegedAccess.Read.AzureResources<br>&#x1F4A1; <a href="https://docs.microsoft.com/en-us/graph/api/resources/privilegedaccess?view=graph-rest-beta" target="_blank">Get privilegedAccess for AzureResources</a><br>If you cannot grant this permission then use parameter <i>-NoPIMEligibility</i></td>
+            </tr>
+          </tbody>
+        </table>
+        Optional: Microsoft Entra ID (AAD) Role 'Directory readers' could be used instead of API permissions (more 'read' than required)
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+Screenshot Azure Portal
+![alt text](img/aadpermissionsportal_4.jpg "Permissions in Microsoft Entra ID")
+
+### PowerShell
```

Comment:
Why have this section duplicated here and in the setup guide?  If you keep it, please make sure it aligns with the final result in the setup guide.